### ISSUE TITLE
Theming Issue

### ISSUE THREAD

**Issue by BeastModePro** [w0lfschild/cDock2#165]
I am unable to use any theme, the theming options are doing nothing on macOS Mojave

**Comment by Davi-T** [comment]
Same here, nothing worky :(

**Comment by rajesh1108** [comment]
Mine also. Want to know if it really work on OS Mojave

**Comment by abuyaqub** [comment]
Same problem... when are you going to fix the problem

**Comment by w0lfschild** [comment]
Unfortunately new security changes in 10.14 (Mojave) prevent cDock from working without System Integrity Protection Disabled.

You may try running it with `csrutil enable --without debug --without fs` to have partial SIP protection.

It is unlikely that there will be a workaround for this change.

**Comment by scottyofeden** [comment]
Transparency still doesn't work even with csrutil disabled. I had paid for cDock when I downloaded High Sierra. Just downloaded Mojave and was asked to pay for cDock again, which I did. But cDock doesn't work with Mojave. I was willing to pay for it if it meant extra work for the developers to get it to work, but to pay for something that doesn't work. The developers need to turn that buy option off.  Thanks!
